macros/models/_sector/transfers/fungible/transfers_fungible_rolling_hour.sql

```diff
@@ -0,0 +1,16 @@
+{% macro transfers_fungible_rolling_hour(transfers_fungible_agg_hour) %}
```

Comment:
Why is there no `transfers_fungible` without aggregation?

i.e. what if I needed to see each transfer event?

Reply:
this is meant to be used for the balances spell. for transfers without aggregation, there's a fungible.transfers spell

Apologies for the confusion!